docs/content/get-started/get-started.md

```diff
@@ -12,41 +12,42 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ```
 
-Get stated guide will provide you quick overview on how to integrate Aperture
-with your application. Once integrated with Aperture, you are ready to enforce
-`rate limiting as a service`.
+Aperture is available as a managed service, [Aperture Cloud][cloud], or can be
+self-hosted within your infrastructure.
 
-Below are two different ways to get started with Aperture, ranging from
-Serverless to Self-Hosted. If you are unsure which one to choose, **we recommend
-starting with the Serverless option**, as it's the fastest way to get started
-and involves fewer teams.
+The easiest way to get started with Aperture is to sign up on Aperture Cloud and
+integrate your application using SDKs.
+
+These are two main modes on how to get started with Aperture.
 
 <Tabs>
 
 <TabItem value="Aperture for Serverless">
 
-This mode is tailored for developers who prefer to use our SDKs directly without
-delving into infrastructure components. It allows developers to focus on their
-application and requires minimal permissions to get started.
+This mode is tailored for developers who prefer to use Aperture SDKs directly
+without delving into infrastructure components. It allows developers to focus on
+their application and requires minimal permissions to get started.
+
+![Aperture Serverless Architecture](./assets/architecture/saas-dark.svg#gh-dark-mode-only)
+![Aperture Serverless Architecture](./assets/architecture/saas-light.svg#gh-light-mode-only)
 
-1. **Sign up for an account**: Get started with Aperture by
-   [creating an account](/get-started/sign-up.md), completing the simple
-   onboarding - giving you an introduction to dashboard - and inviting your team
-   members so you can all collaborate on creating your first policy.
+1. **Sign up for an account**: Get started with Aperture by [creating an
+   account][sign-up], completing the simple onboarding - giving you an
+   introduction to dashboard - and inviting your team members so you can all
+   collaborate on creating your first policy.
 2. **Map to Cloud Agent**: Aperture Agent authenticates requests from SDK
```

Comment:
Instead of "Cloud Agent": "Aperture Cloud"
Instead of "Map to Cloud Agent": "Connect to Aperture Cloud"